### PR TITLE
fix: update questionnaire stepper styling

### LINKED ISSUE #829: Update stepper on eligibility questionnaire.
In light of user feedback, we'd like to update the styling of the stepper on the eligibility questionnnaire. The primary changes are:

1) Increase font size.
2) Change background color to increase contrast.
3) Change font color to increase contrast.

Design is viewable in [Figma](https://www.figma.com/file/wAhoXVmHRIJ2TyBKFVq2JY/Desktop-and-Mobile-Experiences?node-id=3540%3A13497) (03.1).

### PR DESCRIPTION
**Pull request by plunkettgoogle**
## Issue

- Closes #829

## Description

This PR tweaks the styling on the questionnaire stepper to match the [mocks](https://www.figma.com/file/wAhoXVmHRIJ2TyBKFVq2JY/Desktop-and-Mobile-Experiences?node-id=3540%3A13497).

### Desktop
localhost_3000_eligibility_household (2)

### Mobile
localhost_3000_eligibility_household (3)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Prototype/POC (not to merge)
- [ ] This change is a refactor/addresses technical debt
- [ ] This change requires a documentation update
- [ ] This change requires a SQL Script

## How Can This Be Tested/Reviewed?

Navigate to the eligibility questionnaire.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [ ] I have updated the changelog to include a description of my changes
- [ ] I have run `yarn generate:client` if I made backend changes
- [ ] I have exported any new pieces in ui-components

### CODE REVIEW

File: sites/public/styles/overrides.scss [hunk 1]
```diff
@@ -165,6 +165,32 @@ li.progress-nav__item {
...
+  &.is-active:before {
+    @apply bg-primary-dark;
+  }
+  &.is-disabled:before {
+    @apply bg-white;
+    @apply border-2;
+    @apply border-primary;
+  }
+  &:before {
+    background-color: #019682;
```
Comment (GabeBenjamin) [inline sites/public/styles/overrides.scss:178]: Is this color correct? It seems like it's very similar to the primary color?

**Review by GabeBenjamin — CHANGES_REQUESTED** [review]

File: sites/public/styles/overrides.scss (hunk 1 above)
Comment (plunkettgoogle) [reply sites/public/styles/overrides.scss]: I copy-pasted this from _somewhere_, but for the life of me I can't find it in the Figma mocks anymore... :thinking:

File: sites/public/styles/overrides.scss [hunk 2]
```diff
@@ -165,6 +165,32 @@ li.progress-nav__item {
   width: 20%;
 }
 
+.progress-nav__item {
+  &.is-active:before {
+    @apply bg-primary-dark;
+  }
+  &.is-disabled:before {
+    @apply bg-white;
+    @apply border-2;
```
[the remark this answers is not in the document]
Comment (plunkettgoogle) [reply sites/public/styles/overrides.scss]: Done.

File: sites/public/styles/overrides.scss [hunk 3]
```diff
@@ -165,6 +165,32 @@ li.progress-nav__item {
...
+    @apply border-primary;
+  }
+  &:before {
+    background-color: #019682;
+    @apply border-0;
+  }
+  &:after {
+    height: 2px;
+    margin-top: -1px;
```
[the remark this answers is not in the document]
Comment (plunkettgoogle) [reply sites/public/styles/overrides.scss]: The 2px height pushes the line out of center, so the -1px margin pushes it back into center. I'll leave a comment here.

**Review by GabeBenjamin — APPROVED** [review]
LGTM

**Pull request merged**